inc/checks/class-language-update.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace runcommand\Doctor\Checks;
+
+use WP_CLI;
+
+/**
+ * Warns when there are language updates available.
+ */
+class Language_Update extends Check {
+
+	public function run() {
+		ob_start();
+		WP_CLI::run_command( array( 'language', 'core', 'list' ), array( 'format' => 'json' ) );
+		$ret = ob_get_clean();
+		$languages = ! empty( $ret ) ? json_decode( $ret, true ) : array();
+		$update_count = 0;
+		foreach ( $languages as $language ) {
```

Comment:
Instead of having a loop here, you could also use higher-order functions, like so:

```php
$counts = array_count_values( array_column( $languages, 'update' ) );
$update_count = array_key_exists( 'available', $counts ) ? $counts['available'] : 0;
```

I'm not really sure that's better though, as it might be less clear what it actually does.

Reply:
@schlessera 
I'm trying to maintain the coding standards throughout the package. For example, this particular push is similar to `theme-update` check based on its functionality. If it's permitted to ignore the syntactic coherence, then I would like to implement functional programming since it is much neater and reduces the size of the file.